AllReadyApp/Web-App/AllReady/Services/AttachmentService.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Options;
+using Microsoft.Net.Http.Headers;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+using System.Linq;
+
+namespace AllReady.Services
+{
+    public class AttachmentService : IAttachmentService
+    {
+        private const string ContainerName = "attachments";
+        public static string[] AllowedExtensions = new string[] { ".jpg", ".gif", ".bmp", ".pdf" };
```

Comment:
delete `AllowedExtensions`. I don't see anywhere in the original issue that says we only support these type of attachments for a task